test/ListenersOnReconnectTest.js

```diff
@@ -144,11 +147,11 @@ describe('Listeners on reconnect', function () {
                 return Controller.terminateMember(cluster.id, member.uuid);
             }).then(function () {
                 return Controller.startMember(cluster.id);
+            }).then(function () {
+                return Util.promiseWaitMilliseconds(5000);
```

Comment:
Doesn't this wait contradict with the purpose of the test? The client should not lose neither `put` request nor the listener event. Losing the connection to the cluster must not disturb normal operation as long as the cluster comes back alive within connection timeout period. We do not have to give the client a break to get itself together before sending a put.